Makefile

```diff
@@ -233,7 +233,7 @@ node_modules/angular/angular.min.js node_modules/angular-animate/angular-animate
 
 .build/externs/angular-1.3.js:
 	mkdir -p $(dir $@)
-	wget -O $@ https://raw.githubusercontent.com/google/closure-compiler/master/contrib/externs/angular-1.3.js
+	wget -O $@ https://github.com/gberaudo/closure-compiler/raw/master/contrib/externs/angular-1.3.js
```

Comment:
In this case, let's use the initial workaround I proposed and we will change if my PR to closure is merged.